internal/controller/controllers/common.go

```diff
@@ -342,7 +344,7 @@ func isNonePlatformCluster(ctx context.Context, client client.Client, cd *hivev1
 	if err = client.Get(ctx, namespacedName, &clusterInstall); err != nil {
 		return false, !k8serrors.IsNotFound(err), errors.Wrapf(err, "Could not get AgentClusterInstall %s for ClusterDeployment %s", cd.Spec.ClusterInstallRef.Name, cd.Name)
 	}
-	return clusterInstall.Spec.Networking.UserManagedNetworking, false, nil
+	return isUserManagedNetwork(&clusterInstall), false, nil
```

Comment:
maybe worth adding a comment here regarding UserManagedNetworking not always available in spec? or do you think it should be clear enough?

Reply:
This problem seems like it will be short lived, I think the function call is quite self explanatory